en/components/select.md

```diff
@@ -0,0 +1,370 @@
+- When the drop-down list is opened you can navigate to a *specific* item by rapidly typing in the first few characters of the item you wish to go to.
+    - *Note that the speed at which you type in matters.*
+- When the drop-down list is opened, you can navigate through the items using the `Home` and `End` keys.
+- When the drop-down list is opened, navigation with `Up/Down Arrow` keys starts from the selected item if any. Otherwise, it starts from the first item in the list.
+- When the drop-down list is closed you can cycle between its items using the `Up/Down Arrow` keys.
+- When the drop-down list is closed you can also navigate through all items that begin with a specific character, it works the same as if it was opened.
+- When the drop-down list is closed you can also navigate to a *specific* item by rapidly typing in its first few characters. The behaviour is the same as when the drop-down is opened.
+- When the drop-down is closed character key navigation is case insensitive.
+- When the drop-down is closed character key navigation does not change selection when pressing characters that have no matching item(s).
+
+### Item Selection
+An item from the drop-down list can be selected by:
+- Left mouse click.
+- `Enter` key when the respective item is focused.
+- `Space` key when the respective item is focused.
+- Setting the value property in the code.
+- Setting the item's `selected` property.
+- The *first* enabled item in the drop-down list is focused if there is no selected item.
+- The input box is populated with the selected item's value.
+- The input box's text is updated when the selected item changes.
+- The input box is not populated with the text of an item that is focused but not selected.
+- No text is appended to the input box when no item is selected and value is not set or does not match any item.
+- Selection is cleared when setting the value property to a non-existing item.
+- Disabled items are not selectable.
+- Selection is removed if the selected option gets deleted.
+- When there are items with duplicated values, the first one gets selected.
+> It is a good idea to note that `igx-select` supports *single* selection of items only.
+
+### Event emitting
+Since `igx-select` extends `igx-drop-down`, it also makes good use of its events which include:
+
+#### Opened event
+- Emitted when the drop-down is fully opened.
+
+You can make use of the `onOpened` event like so:
+```html
+<igx-select (onOpened)="handleOpened($event)">
+    <igx-select-item [value]="Apple">Apple</igx-select-item>
+</igx-select>
+```
+
+#### Opening/Closing events
+- Emitted on:
+    - input click
+    - select expand/collapse button click (app scenario)
+- Triggered on:
+    - key interaction
+
+The thing about the `onOpening` and `onClosing` events is that they are fired *before* the animation finishes playing, i.e. before the drop-down is fully **opened** or **closed**.
```

Comment:
The thing is not really correct wording. Please reword. One other difference should be, those events are cancel-able I think - if so please mention it